devel/go-tools/Portfile

```diff
@@ -33,25 +34,42 @@ post-extract {
     foreach f [glob -directory ${worksrcpath} *] {
         move ${f}/ ${package_src_dir}
     }
-
 }
 
-set cmds { benchcmp bundle callgraph cover digraph eg fiximports godex godoc
+set cmds { benchcmp bundle callgraph compilebench cover digraph
+           eg fiximports getgo go-contrib-init godex godoc
            goimports gomvpkg gorename gotype goyacc guru html2article
-           present ssadump stress stringer toolstash }
+           present present2md splitdwarf ssadump stress stringer toolstash }
+
+set auth_cmds { authtest cookieauth gitauth netrcauth }
+
+set deps {
+    golang.org/x/net/websocket
+    golang.org/x/crypto/acme/autocert
+    golang.org/x/mod/semver
+    golang.org/x/xerrors
+    golang.org/x/xerrors
+    github.com/yuin/goldmark
+}
 
 build {
-    system -W ${workpath} "GOPATH=${gopath} go get -d golang.org/x/net/websocket"
-    system -W ${workpath} "GOPATH=${gopath} go get -d golang.org/x/crypto/acme/autocert"
+    foreach dep $deps {
+        system -W ${workpath} "GOPATH=${gopath} go get -d ${dep}"
+    }
+    foreach auth $auth_cmds {
+        system -W ${workpath} "GOPATH=${gopath} go build ${package_dir}/cmd/auth/${auth}"
+    }
     foreach cmd $cmds {
         system -W ${workpath} "GOPATH=${gopath} go build ${package_dir}/cmd/${cmd}"
-    }      
+    }
 }
-	
 
 destroot {
+    foreach auth $auth_cmds {
+        xinstall -W ${workpath} ${auth} ${destroot}${prefix}/bin
+    }
     foreach cmd $cmds {
-        xinstall -W ${workpath} ${cmd} ${destroot}${prefix}/bin/
+        xinstall -W ${workpath} ${cmd} ${destroot}${prefix}/bin
     }
```

Comment:
These should be done without `foreach` loops, i.e.:

```
destroot {
     xinstall -W ${workpath} {*}${auth_cmds} {*}${cmds} ${destroot}${prefix}/bin
}
```

Reply:
nice syntax; thank you ryan!